src/index.ts

```diff
@@ -697,17 +714,16 @@ export class KycDao extends ApiBase {
   }
 
   private initNear(blockchainNetwork: NearBlockchainNetwork): void {
-    const errorPrefix = 'Cannot initialize Near SDK';
     if (!blockchainNetwork.startsWith('Near')) {
-      throw new Error(`${errorPrefix} - Not a  Near network: ${blockchainNetwork}`);
+      throw new ConfigurationError(`Not a  Near network: ${blockchainNetwork}`);
```

Comment:
This should be internal, if we call this method on a non-near network that's an SDK bug.